_announcements/week-4.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Week 4 Announcement
+week: 4
+date: 2022-2-13
+---
+
+Hello everyone! Welcome to the Linux Sysadmin Decal! Here are your week 3 announcements:
+
```

Comment:
add an announcement letting students know that if they want additional late labs, they can submit the late lab form + link